test/utils/cryptography/MerkleProof.test.js

```diff
@@ -55,12 +55,12 @@ describe('MerkleProof', function () {
       const merkleTree = StandardMerkleTree.of(toElements('abc'), ['string']);
 
       const root = merkleTree.root;
-      const leaf = merkleTree.leafHash(['a']);
+      const hash = merkleTree.leafHash(['a']);
       const proof = merkleTree.getProof(['a']);
-      const badProof = proof.slice(0, proof.length - 5);
+      const badProof = proof.slice(0, proof.length - 1);
```

Comment:
This `proof` is two hashes long, so `proof.slice(0, proof.length - 1)` makes `badProof` be empty. This is not really meaningful, but a bit more precise.